csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -692,7 +692,7 @@ overrides(
   int base_id: @callable ref);
 
 explicitly_implements(
-  unique int id: @member ref,
+  int id: @member ref,
```

Comment:
I'm not sure I understand the case covered by the quoted comment. Is it the following:
```
using System;

interface IA
{
	int Prop
	{
		get;
	}
}

interface IB : IA
{
}

class C : IB
{
	int IA.Prop // Does this property explicitly implement both IA and IB?
	{
		get
		{
			return 1;
		}
	}

	public int Prop
	{
		get
		{
			return 2;
		}
	}
}

public class Program
{
	public static void Main()
	{
		var c = new C();
		Console.WriteLine(c.Prop); // 2
		Console.WriteLine(((IA)c).Prop); // 1
		Console.WriteLine(((IB)c).Prop); // 1
	}
}
```